src/main/java/teammates/logic/api/Logic.java

```diff
@@ -966,7 +965,7 @@ public boolean hasStudentSubmittedFeedback(FeedbackSessionAttributes fsa, String
         assert fsa != null;
         assert studentEmail != null;
 
```

Comment:
Nit: could add an assert for studentTeam, since the comment is "All parameters are non-null" and the other params have a non-null assertion already

Reply:
I just realized, this method should not even be here. `feedbackSessionsLogic.isFeedbackSessionAttemptedByStudent` already has a facade.